docker-compose.yml

```diff
@@ -55,36 +61,43 @@ services:
       - redis
       - vault
 
+
   # The `worker` compose service hosts the Vela build daemon.
   #
   # This component is used for pulling builds from the FIFO
   # queue and executing them based off their configuration.
   #
   # https://go-vela.github.io/docs/administration/worker/
   worker:
+# comment the 5 lines below to use local server built image
+#    build:
+#      context: ../worker
+#      dockerfile: Dockerfile
+#    container_name: worker
+#    image: worker:local
```

Comment:
i think this is leftover from local testing, i say we remove it

Reply:
Should we leave it in here so that folks can uncomment and run locally?

Reply:
there a similar spot in server section too

Reply:
I'm not a fan because it's beyond the scope of the PR. It also makes some assumptions about local folder structure.  If we want to enhance the file for local dev, I'd do it in separate PR in all the repos that need it.

Reply:
Alright! I'll keep that to a different PR for now to keep this effort going!